plugins/filterFactory_test.go

```diff
@@ -143,48 +143,48 @@ func TestMakeVolumeFilterConfig(t *testing.T) {
 			wantConfig: &VolumeFilterConfig{
 				BaseAssetCapInBaseUnits:  pointy.Float64(3500.0),
 				BaseAssetCapInQuoteUnits: nil,
-				additionalMarketIDs:      nil,
-				optionalAccountIDs:       nil,
+				additionalMarketIDs:      []string{},
```

Comment:
don't recall requesting this change in my previous set of comments. why did these lines change?

Reply:
I sent a message on Slack about this when I sent the PR, but you're right that it should get posted on the PR too. Adding it here:

I had to make a couple changes in other tests that are now failing due to validation. It doesn't really make sense to separate these changes into other PRs, since the changes matter because of validation. These are things like "no nil marketIDs or optionalAccountIDs on volumeFilterConfig after parsing".

Reply:
Oh actually, I did notice these CI failures - that's why I didn't request a review on this two weeks ago - was planning on coming back and fixing those after the other PRs in buy TWAP were done, you just beat me to the review :) The version that you reviewed (the initial commit) has all of these failures.